uint32 seq
apriltagInfos[] bundle
